### PR TITLE
Support for importing from a network URL

### PR DESCRIPTION
**Pull request by ZappoMan**
* added support in Octree to read an SVO from a network resource
* added Import Entities from URL to editEntities.js

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: feddea821ad4d144e5754dcd807114a090b3cc08 available at: https://gist.github.com/781ffd9091e067f2225b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4406.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4406.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e16961c958244368a7dfe06a5f06565f2129bd83 available at: https://gist.github.com/781ffd9091e067f2225b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4406.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4406.dmg

File: libraries/octree/src/Octree.cpp [hunk 1]
```diff
@@ -1836,141 +1844,185 @@ int Octree::encodeTreeBitstreamRecursion(OctreeElement* element,
...
-        
-        unsigned long headerLength = 0; // bytes in the header
-        
-        bool wantImportProgress = true;
+        emit importProgress(100);
+        file.close();
+    }
+    
+    return fileOk;
+}
```
Comment (AndrewMeadows) [inline libraries/octree/src/Octree.cpp]: fileLength declaration should go inside if(isOpen) block.

Comment (ZappoMan) [inline libraries/octree/src/Octree.cpp]: I have changed this... but github isn't showing your comment as outdated.

**Comment by hifi-gustavo** [comment]
Build console output for commit: b9671f3f9b2afc8597f6e98f9f08c67bdff24559 available at: https://gist.github.com/781ffd9091e067f2225b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4406.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4406.dmg

**Pull request merged**